components/post_view/commented_on_files_message/commented_on_files_message.jsx

```diff
@@ -17,6 +17,19 @@ export default class CommentedOnFilesMessage extends React.PureComponent {
          * An array of file metadata for the parent post
          */
         fileInfos: PropTypes.arrayOf(PropTypes.object),
+        actions: PropTypes.shape({
+
+            /*
+             * Function to get file metadata for a post
+             */
+            getFilesForPost: PropTypes.func.isRequired,
+        }).isRequired,
+    }
+
+    componentDidMount() {
+        if (!this.props.fileInfos || this.props.fileInfos.length === 0) {
```

Comment:
Is this called when the post doesn't have any files at all? We might need to look at `post.file_ids` in that case

Reply:
This is from a selector `makeGetFilesForPost` fed as `fileInfos ` This is how it used to be before removing for post metadata so pretty much copying old logic here.